### ISSUE TITLE
Project Tasks

### ISSUE THREAD

**Issue by Samantha-Mcguigan** [Samantha-Mcguigan/newsAnalysis#12]
- [ ] Selection principle for sources. @jonhoranic
- [x] Topics headings on homepage. @jonhoranic
- [ ] Finish network analysis and post on site @Samantha-Mcguigan 
- [x] Write up short biographies for the about page.
- [ ] Add a link to this GitHub repo to the site.

**Comment by Samantha-Mcguigan** [comment]
@jonhoranic @ajnewton1 We can distribute tasks in this issue, just edit the the the comment above.

**Comment by dotfig** [comment]
@jonhoranic merger conflict is resolved

**Comment by RJP43** [comment]
@ebeshero and I will investigate **Key Word in Context** to create a table of the phrases for the top colocated words so we can have some context with how the different networks/articles/topics are using those shared words.

**Comment by dotfig** [comment]
I am going to be working on the Analysis page so try not to work on that @Samantha-Mcguigan @jonhoranic

**Comment by dotfig** [comment]
I am done messing with the Analysis page. Feel free to pull and edit. @Samantha-Mcguigan @jonhoranic

**Comment by dotfig** [comment]
Also, I am not the best with grammar (I am actually very bad) so don't be afraid to double check my descriptions for the graphs as well as the about page.

**Comment by ebeshero** [comment]
@ajnewton1 The new graph looks great. But why do you say on the page that the first graph (I think?) is not a stacked bar plot?

@jonhoranic @Samantha-Mcguigan @RJP43 Here's the eXist-db documentation on their kwic (keyword in context) module: It's pretty straightforward to use!
http://exist-db.org/exist/apps/doc/kwic.xml

**Comment by jonhoranic** [comment]
@ajnewton1 I will make an attempt to look over the site after I take care of this paper I have due tomorrow. I do not know what we might do to fill out some of the other pages, I think we can come up with something for both the organizations and the topics, but if you or @Samantha-Mcguigan have ideas let me know.

**Comment by dotfig** [comment]
@ebeshero The graphs, in my opinion, are a little confusing to read. I looks like the bars are stacked on top of each other. For example on the last graph on the page, it looks like the democrats are referenced less than the republicans are referenced because there is a lot of red and a little amount of blue. However, it is the other way around and also the black bar doesn't necessarily represent the "undocumented" in the first graph. 
So to summarize, the graphs are terrible...

**Comment by ebeshero** [comment]
@ajnewton1 Aha! So what you're saying is...these graphs need some help, and something's wrong with how they're rendering...Would you like some help with debugging them?

One suggestion to make the data appear to be more evenly comparable in the graphs is to try plotting their data as percentages...We could try that, too, and get a sense of how that looks.
